llumnix/entrypoints/bladellm/eas_serve.py

```diff
@@ -0,0 +1,83 @@
+import signal
+import subprocess
+import sys
+import os
+import time
+import ray
+
+from llumnix.logging.logger import init_logger
+from llumnix.entrypoints.setup import connect_to_ray_cluster
+
+logger = init_logger("llumnix")
+
+connect_to_ray_cluster()
+
+
+@ray.remote
+class EasLaunchActor:
+    def __init__(self):
+        self.process = None
+
+    def launch_serve(self, command):
+        # avoid "RuntimeError: No CUDA GPUs are available"
+        os.environ["CUDA_VISIBLE_DEVICES"] = "0"
```

Comment:
Can we pass this environment variables to subprocess, rather than set it in parent process?